src/modules/move.js

```diff
@@ -104,7 +104,16 @@ module.exports = bot => {
 
     bot.editChannel(thread.channel_id, {
       parentID: targetCategory.id
-    }).then(() => syncThreadChannel(threadChannel, targetCategory));
+    }).then(() => { 
+		syncThreadChannel(threadChannel, targetCategory)
+	}).then(() => {
+		bot.createMessage(threadChannel.id, {
+			content: `<@%${config.inboxAdminRoleId}>, a thread has been moved.`,
+			allowedMentions: {
+				roles: true,
+			},
+		});
+	});
```

Comment:
Indentation looks a lil off, is that GitHub or did you forgot to do that?

Reply:
Editor was still using the Dyno standard, but changed for readability and consistency